package.json

```diff
@@ -21,5 +23,21 @@
     "react-dom": "18.2.0",
     "tailwindcss": "3.3.2",
     "typescript": "5.1.6"
+  },
+  "devDependencies": {
+    "@jest/globals": "^29.6.1",
```

Comment:
Delete it with `npm uninstall @jest/globals` because these things are shipped with `jest` directly